src/lib/ui/thermostat/Temperature.tsx

```diff
@@ -0,0 +1,26 @@
+interface TemperatureProps {
+  fahrenheit: number
+  celsius: number
+  unit: 'fahrenheit' | 'celsius'
+  hideUnit?: boolean
+}
+
+export function Temperature({
+  fahrenheit,
+  celsius,
+  unit,
+  hideUnit,
+}: TemperatureProps): JSX.Element {
+  const temperature = unit === 'fahrenheit' ? fahrenheit : celsius
+  const degree = unit === 'fahrenheit' ? t.degreeFahrenheit : t.degreeCelsius
+  return (
+    <span>
+      {Math.trunc(temperature)}°{hideUnit != null && degree}
```

Comment:
We should use this function anywhere we display temperature: https://github.com/seamapi/react/blob/31b83fff25332bb620070305c67ec935a91f233c/src/lib/ui/thermostat/ClimateSettingStatus.tsx#L88